### PR TITLE
[ko] Update outdated files in dev-1.22-ko.1 (p6)

### PR DESCRIPTION
**Pull request by jihoon-seo**
Related issue: 
- #29257

> 19. [ ]  M19.  `content/en/docs/concepts/services-networking/dns-pod-service.md`  | 23(+S) 1(-)
> : Also in #29253 
> 20. [ ]  M20.  `content/en/docs/concepts/services-networking/ingress-controllers.md`  | 1(+XS) 0(-)
> 21. [ ]  M21.  `content/en/docs/concepts/services-networking/ingress.md`  | 6(+XS) 1(-)
> : Also in #29253 
> 22. [ ]  M22.  `content/en/docs/concepts/services-networking/network-policies.md`  | 11(+S) 3(-)
> : Also in #29253 
> 23. [ ]  M23.  `content/en/docs/concepts/services-networking/service.md`  | 43(+M) 9(-)
> : Also in #29253 
> 24. [ ]  **M24.**  `content/en/docs/concepts/storage/persistent-volumes.md`  | **118(+L) 32(-)**
> : Also in #29253 

/language ko

### CODE REVIEW

**Review by ClaudiaJKang — COMMENTED** [review]
/lgtm 
@jihoon-seo 수고하셨습니다!!

File: content/ko/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -785,6 +795,82 @@ spec:
       storage: 10Gi
 ```
 
+## 볼륨 파퓰레이터와 데이터 소스
```
Comment (seokho-son) [inline content/ko/docs/concepts/storage/persistent-volumes.md:798]: 한영병기하면 어떨까요?

Comment (jihoon-seo) [reply content/ko/docs/concepts/storage/persistent-volumes.md]: 좋은 의견 감사합니다. 반영하였습니다. 😊

**Comment by seokho-son** [comment]
감사합니다!!
/lgtm
/approve

**Pull request merged**